### PR TITLE
Pcm 1811 PCM 1848

### PR DESCRIPTION
**Pull request by sshumake**
@renujhamtani @vrathee @gandhikeyur

### CODE REVIEW

File: app/cases/controllers/new.js [hunk 1]
```diff
@@ -144,7 +171,7 @@ angular.module('RedhatAccess.cases').controller('New', [
                 AlertService.addStrataErrorMessage(error);
             });
             if (window.chrometwo_require !== undefined) {
-                breadcrumbs = [
+                var breadcrumbs = [
```
Comment (gandhikeyur) [inline app/cases/controllers/new.js]: var breadcrumbs, will this work in QA?

**Comment by sshumake** [comment]
@gandhikeyur nice catch! I have pushed a fix.

**Comment by gandhikeyur** [comment]
Looks good. Reviewed!!

**Comment by vrathee** [comment]
Reviewed!

**Pull request merged**